atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -138,7 +138,7 @@
 
 @SuppressWarnings("checkstyle:all")
 public class SnapshotTransactionTest extends AtlasDbTestCase {
-    private static final TransactionConfig TRANSACTION_CONFIG = ImmutableTransactionConfig.builder().build();
+    private static TransactionConfig transactionConfig = ImmutableTransactionConfig.builder().build();
```

Comment:
Let's not make this static if it's mutable. Just set it to the correct value in each test or in the `@before` method and then update the value for the specific test that needs something different.

As is, the order that tests run in could affect results.